crates/modelardb_server/src/storage/mod.rs

```diff
@@ -95,6 +96,8 @@ const APACHE_PARQUET_FILE_SIGNATURE: &[u8] = &[80, 65, 82, 49]; // PAR1.
 /// than the requested due to internal alignment when allocating memory for the two array builders.
 const UNCOMPRESSED_DATA_BUFFER_CAPACITY: usize = 64 * 1024;
 
+const TRANSFER_BATCH_SIZE_IN_BYTES: usize = 64 * 1024 * 1024; // 64 MiB;
```

Comment:
A doc comment should be added to this `const` like is already done for the other `const`s.

Reply:
Fixed.